docs/schedule.rst

```diff
@@ -69,3 +69,21 @@ Schedule items which are not talks have ``talk-type-none`` as the CSS class.
 
 A per item CSS class can also be set using the ``css_class`` attribute on the
 schedule item.
+
+
+Adding additional schedule validation
+=====================================
+
+Wafer runs validation on the slots and the schedule items. This behaviour
+can be extended by providing custom validators.
+
+Each validator is called with a list of all the items and is expected to
```

Comment:
Should we explain that item validators get a list of items and slot validators get a list of slots?